index.js

```diff
@@ -28,16 +30,16 @@ store.subscribe(() => {
 })
 
 store.dispatch({ type: 'ADD_BOOK', title: 'Война и мир, Voïna i mir' })
-assert.deepStrictEqual({books: new Set().add('Война и мир, Voïna i mir'), customers: new Set()}, store.getState())
+assert(Immutable.is(Map({books: Set().add('Война и мир, Voïna i mir'), customers: Set()}), store.getState()) === true)
```

Comment:
There must be a simpler way to test it, maybe by using `assert.deepStrictEqual()` and `state#toJS()`?